src/main/resources/features/product_detail_page/see_size_guide.feature

```diff
@@ -0,0 +1,9 @@
+Feature: See Product Size Guide
+  The user should be able to see a size guide for all categories
```

Comment:
Can you please specify more what do you mean by category? My first understanding was it is about SPHERE categories, but you mean T-shirts and shoes.